### DISCUSSION TITLE
Is there any way to change duration by variables?

### DISCUSSION THREAD

**Discussion by jiwooIncludeJeong** [vydimitrov/react-countdown-circle-timer#73] [Q&A]
I want to change duration of timer by user entered value.
Is there any way to do like this?

**Comment by vydimitrov** [comment]
Hi, sure thing, you will need to pass a new value for the `key` prop every time when the duration changes. Like in this demo - https://codesandbox.io/s/tender-bogdan-qd35m

**Reply by jiwooIncludeJeong** [reply]
Thanks a lot! I did a lot of ways but it didn't work... 
The problem was the variable was not a Number..... lol :) Thanks!

**Reply by vydimitrov** [reply]
TypeScript helps here :)

**Reply by batuhanbag** [reply]
I did tons of ways but I didn't work. I wish firstly search issues a little bit :). Thanks a lot @vydimitrov
